src/main/java/com/balloon_spring_jpa/balloon/entity/Order.java

```diff
@@ -0,0 +1,69 @@
+package com.balloon_spring_jpa.balloon.entity;
+
+import com.balloon_spring_jpa.balloon.balloonEnum.OrderStatus;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Setter
+@Getter
+@Table(name = "customer_order")
+public class Order {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "address")
+    private String address;
+
+    @Column(name = "total_price")
+    private BigDecimal totalPrice;
+
+    @Column(name = "date_time", columnDefinition = "TIMESTAMP", updatable = false)
+    private LocalDateTime dateTime;
+
+    @Enumerated(value = EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    @ManyToOne(cascade = CascadeType.REMOVE)
+    private Customer customer;
+
+    @OneToMany(mappedBy = "order", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private List<LatexBalloonQuantityInOrder> latexBalloonQuantity;
+
+    @OneToMany(mappedBy = "order", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private List<FoilBalloonQuantityInOrder> foilBalloonQuantity;
+
+    @PrePersist
```

Comment:
why not just make it default value for dateTime?

Reply:
cause i don't know how it do with annotation only(but i understand that it have to be with it). i tried many options with annotations but it doesn't works (always  " = null"). I need help with it.